docs/api_docs/clustering.md

```diff
@@ -1,18 +1,15 @@
 ---
 tags:
   - API
-  - Clustering
+  - clustering
```

Comment:
the old file is now linker_clustering.md to distinguish from the 'plain' linker method